articles/precarious-data.md

```diff
@@ -1,51 +1,217 @@
+The alternative is to identify content by its "fingerprint" rather than identifying it by its location. That way, when someone says "Look at the content with this fingerprint" you can get it from anyone who has a copy of the content. To do this, we identify content by its cryptographic hash. When we identify content in this way, using the content's fingerprint instead of its location to identify it, this is called _content-addressing_.
+The cryptographic hash for a piece of content never changes, which means **content addressing guarantees that the links will always return the same content**, regardless of _where_ I retrieve the content from.  You can do it by using a _content-addressed_ protocol like IPFS instead of using a _location-addressed_ protocol like HTTP.
+
+This decentralized approach radically increases the durability of data. It ensures that data will not become endangered as long as anyone is still relying on it because anyone can hold a valid copy of the data they care about. If you hold a copy of a dataset on any of your devices you become part of the network of stewards who protect if from being lost. You won't have to worry about whether someone is going to turn off the servers where your data are hosted because _you are one of the hosts_. You and your peers hold the data among yourselves and are able to share the data directly with each other without relying on centralized points of failure.
 
-The cryptographic fingerprint for a piece of content never changes, which means **content addressing guarantees that the links will always return the same content**, regardless of _where_ I retrieve the content from.
+Decentralization also increases the integrity of data because links are content-addressed. This means we can validate data by checking the data's fingerprints against the links. That kind of validation is impossible with location-addressed links. This is especially powerful on the large scale, where thousands of datasets reference each other millions of times. With location-addressed links, all of those connections are brittle. With content-addressed links, the connections become resilient.
 
-This is done using a _content-addressed_ protocol like IPFS instead of using a _location-addressed_ protocol like HTTP.
 
 ## How to Do It: Write Content onto IPFS and Publish the Hashes
 
-If you want to use IPFS to make content available in a decentralized fashion, you need to install IPFS, add your content to IPFS, and then tell your peers the cryptographic hashes (aka fingerprints) for the content you added. They can use those hashes to retrieve copies of the content. They can also use those hashes to link to the content.
+If you want to use IPFS to save your data, you need to write your content to IPFS and then replicate that content across your network of peers using the IPFS protocol. To do this, there are four main steps:
```

Comment:
Explain what IPFS is. You haven't until now. A paragraph or two should do it. What does it stand for, how does it help with content addressing, how do I find other nodes on it, who is developing it, and so on.